include/dlaf/util_matrix.h

```diff
@@ -0,0 +1,44 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include <exception>
+#include <string>
+
+/// @file
+
+namespace dlaf {
+namespace util_matrix {
+
+/// @brief Verify if dlaf::Matrix is square
+///
+/// @param M refers to a dlaf::Matrix object
```

Comment:
Sorry, my comment was not clear.

I think that with `M` you are referring to the template parameter `Matrix`.

i.e. `/// @tparam Matrix refers to a dlaf::Matrix object`

Same applies to the documentation of next function.

Reply:
Nope, the comment was clear. My fault, sorry.